client/src/components/jokeList.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+
+import { getJokes } from "../actions";
+import "./jokes.css";
+
+class jokeList extends Component {
+  componentDidMount() {
+    this.props.getJokes();
+  }
+  render() {
+    return (
+      <div className="joke-container">
+        {this.props.jokes.map(joke => (
+          <p key={joke.id}>{joke.joke}</p>
```

Comment:
might want to re-evaluate "joke.joke" lol.. using the same word some times can get pretty weird at times.